dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm

```diff
@@ -1741,7 +1741,7 @@ EOF
   #      XSINTERFACE_FUNC_SET(cv, $value);
 
   print Q(<<"EOF") if defined $self->{XsubAliases} or defined $self->{interfaces};
-#    {
+#    [[
```

Comment:
I'm not familiar enough with C to understand why you are suggesting replacing single braces with double brackets.  Can you explain?

Reply:
That's an escaped string. The 'Q' function converts [[ to { and ]] to }. So in the final emitted C code it is still a single brace, but in the perl source the unbalanced brace is hidden from text editors. This is already done many times in ParseXS.pm; this commit just changes the few places where it's not done.